CMakeLists.txt

```
@@ -63,6 +63,7 @@ endif()
 set("dev" OFF CACHE BOOL "The build will only be used for development: no global installation, run from the source directory, compile helper utilities")
 set("WITH_CLI11" OFF CACHE BOOL "Use CLI11 instead of bundled tclap")
 set("ENABLE_MAGE" ON CACHE BOOL "Build with MAGE")
+set("ENABLE_SONIC" ON CACHE BOOL "Build with libsonic.")
```

```suggestion
set("ENABLE_SONIC" OFF CACHE BOOL "Build with libsonic.")
```

lets disable it by default as in scons.

---

Done.